constants.js

```diff
@@ -0,0 +1,79 @@
+const SITE_TITLE = 'Awe & Reverence';
```

Comment:
I'm sorry for the miscommunication, could we put it back to the `constants/` subdirectory? (So, keep `constants/skills.js` and `constants/nav_links.js`)

And for the new ones like `SITE_TITLE` and `PAGE_TITLES`, we can make a `constants/seo.js` or `constants/general.js` file

Reply:
We re-added the constants subdirectory, and moved the constants to their proper files.